README.md

```diff
@@ -106,22 +106,14 @@ yarn run build --commit=HEAD --network=gaia-7005
 
 You can specify `--help` to see all options with explanations.
 
-Unzip the app:
+To unzip the app:
 
 ```bash
 open builds/Voyager/Cosmos_Voyager-v0.9.4-Linux.tar.gz
 open builds/Voyager/Cosmos_Voyager-v0.9.4-macOS.zip
 open builds/Voyager/Cosmos_Voyager-v0.9.4-Windows.zip
 ```
 
-Then open the Cosmos Voyager.app:
-
-```bash
-open builds/Voyager/Cosmos_Voyager-v0.9.4-Linux/Cosmos\ Voyager.app
```

Comment:
^^ the simple solution

Reply:
the documentation was out of date. i tried to update it - but i think it's unnecessary.